RELEASES.md

```diff
@@ -1,3 +1,207 @@
+Version 1.63.0 (2022-08-11)
+==========================
+
+Language
+--------
+- [Remove migrate borrowck mode for pre-NLL errors.][95565]
+- [Modify MIR building to drop repeat expressions with length zero.][95953]
+- [Remove label/lifetime shadowing warnings.][96296]
+- [Allow explicit generic arguments in the presence of `impl Trait` args.][96868]
+- [Make `cenum_impl_drop_cast` warnings deny-by-default.][97652]
+- [Prevent unwinding when `-C panic=abort` is used regardless of declared ABI.][96959]
+- [lub: don't bail out due to empty binders.][97867]
```

Comment:
I would like to have a more accessible description for this "lub" change, but I don't even understand it myself. :sweat_smile:

@lcnr can you help with that?

Reply:
from zulip:
> Boxy: returning fn ptrs with differing lifetimes from match arms will now correctly subtype to a return type instead of erroring

probably best to confirm with lcnr though :thinking:

Reply:
Discussed on that Zulip thread - https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler/topic/release.20notes.20help -- my sense is that there's not much value in trying to improve this text and we might even drop it entirely. For the time being I'm going to go ahead and approve this PR so we can get the release notes landed prior to promotion kicking off.